src/app/authorization/registration/registration.component.html

```diff
@@ -1,129 +1,238 @@
+        <div class="row">
+          <div class="field">
+            <div class="m-1">Gender</div>
+            <div class="wrapper">
+              <div class="form-check form-check-inline">
+                <input
+                  type="radio"
+                  class="form-check-input"
+                  value="male"
+                  id="male"
+                  formControlName="gender"
+                />
                 <label class="form-check-label" for="male">Male</label>
-                
-            </div>
-            <div class="form-check form-check-inline">
-                <input type="radio" class="form-check-input" value="f" id="female" formControlName="gender">
+              </div>
+              <div class="form-check form-check-inline">
+                <input
+                  type="radio"
+                  class="form-check-input"
+                  value="female"
+                  id="female"
+                  formControlName="gender"
+                />
                 <label class="form-check-label" for="female">Female</label>
-            </div>
-            <div class="form-check form-check-inline">
-                <input type="radio" class="form-check-input" value="o" id="other" formControlName="gender">
-                <label class="form-check-label" for="female">Other</label> 
-            </div>
-            <div class="form-text text-danger" *ngIf="Gender.errors?.['required'] && (Gender.dirty || Gender.touched)">
+              </div>
+              <div class="form-check form-check-inline">
+                <input
+                  type="radio"
+                  class="form-check-input"
+                  value="other"
+                  id="other"
+                  formControlName="gender"
+                />
+                <label class="form-check-label" for="female">Other</label>
+              </div>
+              <div
+                class="form-text text-danger"
+                *ngIf="registrationFormValidator['gender']?.errors?.['required'] && (registrationFormValidator['gender']?.dirty || registrationFormValidator['gender']?.touched)"
+              >
                 Gender is required.
+              </div>
             </div>
+          </div>
         </div>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-mb-3">
-            <label for="birthday"> Date Of Birth :</label>
-            <input type="date" style="margin-top: 10px;" formControlName="birthday" [max]="getToday()">
-            <div class="form-text text-danger" *ngIf="Birthday.errors?.['required'] && (Birthday.dirty || Birthday.touched)">
+        <div class="row">
+          <div class="col-mb-3">
+            <div class="field">
+              <label for="birthday" class="form-label"> Date Of Birth :</label>
+              <input
+                type="date"
+                class="form-control input-sm"
+                formControlName="birthday"
+                [max]="getToday()"
```

Comment:
Instead of calling function here can we make one variable and assign today date there and use it here ?